test/utility_tests.jl

```diff
@@ -3,22 +3,27 @@ using StochasticDiffEq, LinearAlgebra, SparseArrays, Random, Test, DiffEqOperato
 
 @testset "Derivative Utilities" begin
   @testset "WOperator" begin
-    Random.seed!(0); y = zeros(2); b = rand(2)
-    mm = I; _J = rand(2,2)
-    W = WOperator(mm, 1.0, DiffEqArrayOperator(_J))
-    set_gamma!(W, 2.0)
-    _W = mm - 2.0 * _J
-    @test convert(AbstractMatrix,W) ≈ _W
-    @test W * b ≈ _W * b
-    mul!(y, W, b); @test y ≈ _W * b
+    Random.seed!(123)
```

Comment:
I don't think we need those tests at all, since the code is centralized in DiffEqBase.jl. We should put those in DiffEqBase.jl instead.

Reply:
No the code for functions like `calc_W` is not centralized right now because of small differences like in StochasticDiffEq, `update_W!` returns J as its value is used. Should I move them to DiffEqBase in this PR?